react-design/my-app/src/Buttons.js

```diff
@@ -0,0 +1,3 @@
+export default function Buttons({ text, onCustomClick }) {
+  return <button onClick={onCustomClick}>{text}</button>;
+}
```

Comment:
I suggest renaming this component: `Buttons` implies this component contains a list of buttons, not exactly one button, which can be misleading and make it harder to debug later.